### ISSUE TITLE
Sept 2023 Contribution - {ChrisBck#1751}

### ISSUE THREAD

**Issue by chrisbck** [Phala-Network/growth-program#268]
### A.	Personal info:
Ambassador Rank: Ambassador
Telegram Username: [@chrisbck](https://github.com/chrisbck)
Twitter Url: https://twitter.com/BckChris
PHA Wallet Address: 43ZKJ2DEBagpMyn4EQvTu7niH1xRYGvfCJyGZYZyW6STFCbd
Departments: CC/CM
Department Score Weight: 70/30

### CC:
Began a series of articles called “Big Phat Guide for Phala Network”. Inspired by the “Dummies Guide for ….” books, They are written jargon free and in an informal tone. Approximately 2000 words each.
- Part One - https://medium.com/@ChrisBck/big-phat-guide-for-phala-network-part-one-80be3a4462fc
- Part Two - https://medium.com/@ChrisBck/big-phat-guide-for-phala-network-part-two-cfa7e2a8137f

Overall was not a significant amount of content released, I spent a lot of time prototyping a potential community project that I have requested the team’s approval before putting it out.

### CM:
- responded to PW tickets #0551 and #0552, closed out #0450
- Didn't have much community engagement this month. Mainly responding to queries in PhalaWorld.

### ART: 
Only output one image to accompany the articles I wrote.
<img width="480" alt="BPG Title" src="https://github.com/Phala-Network/growth-program/assets/32775411/f9856c5b-6bfb-4dc9-a4e6-5601e6c06382">

In summary it has not been a very productive month for me as have been preoccupied with other events that have kept me busy, I do have plans for October though, which should be a better month, especially if I get the go ahead for the PW community project.

### D. Suggested scores: 
 
- CM 15/24
- CC 28/56
 
- ART 1/10

 
Overall score: 44/100.
